CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 with the exception that 0.x versions can break between minor versions.
 
 ## Unreleased
+### Added
+- Support for [conditionals](https://www.regular-expressions.info/conditional.html).
```

Comment:
Can you add a small example (regex plus some text it matches against) here that gives an idea of what this is about?

Reply:
good idea, done. I somehow missed that we had done that for previously added features, sorry :)